db/schema.rb

```diff
@@ -11,7 +11,7 @@
 #
```

Comment:
You need to make the schema for Postgres and Oracle as well. `make schema` for Oracle IIRC